### ISSUE TITLE
Cannot parse multipoint

### ISSUE THREAD

**Issue by rogue780** [geo-io/wkb-parser#3]
Here is the WKB:

```
0104000020E61000000700000001010000000000000000C050C000000000000022C0010100000000000000000037C00000000000804A4001010000000000000000C05440000000000000334001010000000000000000003640000000000080524001010000000000000000002C400000000000C054C0010100000000000000006062C00000000000004640010100000000000000008042C00000000000804BC0
```

The error I get is:

  [GeoIO\WKB\Parser\Exception\ParserException]  
  Parsing failed: SRID mismatch between "2D" and expected "2D".  

in postgres, I get the following:

```
                          st_astext                           |                                                                                                                                                           spatial_data                                                                                                                                                           
--------------------------------------------------------------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
 MULTIPOINT(-67 -9,-23 53,83 19,22 74,14 -83,-147 44,-37 -55) | 0104000020E61000000700000001010000000000000000C050C000000000000022C0010100000000000000000037C00000000000804A4001010000000000000000C05440000000000000334001010000000000000000003640000000000080524001010000000000000000002C400000000000C054C0010100000000000000006062C00000000000004640010100000000000000008042C00000000000804BC0
```

Removing 

```
        if ($this->srid && $srid !== $this->srid) {
            throw new \RuntimeException(sprintf(
                'SRID mismatch between %s and expected %s.',
                json_encode($dimension),
                json_encode($this->dimension)
            ));
        }
```

from Parser.php eliminates the problem, but I'm guessing that is less than ideal.

Please let me know if you need more information.

Thanks!

**Comment by jsor** [comment]
Thanks for the report. This is fixed in 34d6e9dd813cc5a28dbc9b4e6ce73f60eec78a07. I've tagged v1.0.1 which contains the fix.

**Comment by rogue780** [comment]
@jsor Awesome, thanks! Works great now.
